CMakeLists.txt

```diff
@@ -3,7 +3,9 @@ project(GOESP)
 
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_C_FLAGS_RELEASE "-O3 -s -fvisibility=hidden -flto -fwhole-program -DNDEBUG")
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -fno-rtti")
+set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} --no-gnu-unique -fno-rtti")
+
+set(CMAKE_BUILD_TYPE RELEASE)
```

Comment:
I wouldn't do that, because it makes impossible to set `CMAKE_BUILD_TYPE` from outside.